docs/data/sources/shopify.md

```diff
@@ -5,66 +5,39 @@ description: Amplitude’s app with Shopify is a smart analytics app that automa
-3. Go to your Amplitude project, then navigate to **Sources and Destinations > Destinations**.
-4. Under **Add More Destinations …**, click **Shopify**.
-5. Copy the Amplitude project’s API key.
-6. In the Shopify admin portal, enter the API key in **Amplitude API Key** field. Then click **Connect**.
-7. Choose a `User_ID` for known customers. To support a broader range of use cases, the app lets you choose which of the following fields you want to send as the `User_ID` for known customers.
-   - **Email (default)** – Recommended when other platforms use the email and can’t hash it, and you are comfortable with the privacy implications.
-   - **Hashed email** – The MD5 email hash is useful if you have other marketing platforms sending traffic where you know the email of the visitor (e.g. email marketing like Bronto or Marketo), but not their Shopify customer ID.
-8. Finally, navigate to **Themes > Current Theme > Actions > Edit Code**, and check if the Amplitude code snippet is installed. It should be located before the theme’s closing `</head>` tag in the `theme.liquid` file. If not, add it.
 
-    ```liquid
-    <!-- Amplitude -->
-    {% render 'amplitude' %}
-    ```
-
-## SDK Configurations
-
-Amplitude's JS SDK powers the out-of-the-box integration with Shopify stores. This behavior can be extended with custom-defined events and SDK configurations.
-
-To do this perform the following steps:
-
-1. In your Shopify Admin portal, navigate to **Online Store > Themes > Actions > Edit Code**.
-2. Create an `amplitude-options.liquid` file in the Layout section.
-3.  Insert the following code template inside this file:
-  
-    ```json
-    window.amplitudeShopify.customOptions = {
-      // custom options definitions
-    }
-    ```
-
-!!!note
-    Any configurations defined here are passed onto the underlying JS SDK used by Amplitude's Shopify app (as this also overrides the default options you see in `theme.liquid`).
+## Key Limitations
+1) Each shopify store uses a particular theme to shape the online store experience for merchants and their customers. Currently Amplitude's Shopify App only installs this client-side snippet code to every existing theme in a store. However, if a customer was to add a new theme after the installation, it would mean the Amplitude code snippet would not be installed and certain client-side events will not be forwarded from Shopify to Amplitude. 
+2) Currently Amplitude's Shopify App code is not open-sourced which means we are not able to support specific customer use cases (such as custom event properties).
+3) User identity User_ID is limited to just email and SHA-256. You are not able to customize or set your own identifier (User_ID).
+4) Amplitude’s Shopify app does not track revenue event properties.
+5) Amplitude’s Shopify app does not track Order Fulfillment status events.
+6) Amplitude’s Shopify app does not track Subscription events.
+7) Amplitude’s Shopify app does not track sales/return events.
```

Comment:
```suggestion
- The app doesn't sales/return events.
```

Reply:
Is sales/return a specific event type? Or is this Sales and Return events? We should be specific.